tiled/_tests/test_queries.py

```diff
@@ -168,6 +171,9 @@ def cm():
         cm = nullcontext
     with cm():
         assert list(client.search(FullText("z"))) == ["z", "does_contain_z"]
+        # plainto_tsquery fails to find certain words, weirdly, so it is a useful
+        # test that we are using tsquery
+        assert list(client.search(FullText("purple"))) == ["full_text_test_case"]
```

Comment:
I don't know how much to expect of this index, but could we do partial word search? e.g. `urple`

Reply:
I think support is limited, but robust fuzzy text search is next up for @Kezzsim.